### PR TITLE
updated tr_tr

### PR DESCRIPTION
**Pull request by Hexasan**
+updated turkish language file

that old one was a mess, lowercase things that should be uppercase, things already translated in the base game that is translated differently here, other weird things, but I believe I fixed most of them

### CODE REVIEW

**Comment by Hexasan** [comment]
netherite sickle was translated as "dutch sickle" in the old translation lmao

File: src/main/resources/assets/prefab/lang/tr_tr.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
     "item.prefab.item_basic_structure": "Basit Yapıt",
     "item.prefab.item_instant_bridge": "Anında Köprü",
     "item.prefab.item_moderate_house": "Orta Boy Ev",
-    "item.prefab.item_bulldozer": "Ekskavatör",
+    "item.prefab.item_bulldozer": "Yaratıcı Buldozer",
```
Comment (Brian-Wuest) [inline src/main/resources/assets/prefab/lang/tr_tr.json]: The bulldozer isn't just used in creative-mode. This should just translate from the English word of "Bulldozer"

File: src/main/resources/assets/prefab/lang/tr_tr.json [hunk 2]
```diff
@@ -92,17 +92,17 @@
...
-    "block.prefab.block_dirt_stairs": "Kirli Merdivenler",
-    "block.prefab.block_grass_stairs": "Çim Merdivenler",
-    "block.prefab.block_dirt_slab": "Kir Levha",
-    "block.prefab.block_grass_slab": "çim levha",
-    "block.prefab.block_dirt_wall": "Kir Duvar",
-    "block.prefab.block_grass_wall": "çim duvar",
+    "block.prefab.block_dirt_stairs": "Toprak Merdivenler",
+    "block.prefab.block_grass_stairs": "Çimenli Toprak Merdivenler",
+    "block.prefab.block_dirt_slab": "Toprak Basamak",
+    "block.prefab.block_grass_slab": "Çimenli Toprak",
```
Comment (Brian-Wuest) [inline src/main/resources/assets/prefab/lang/tr_tr.json]: This translates to "Grassy Soil" and is used for both the Grass Slab and Grass Wall.

File: src/main/resources/assets/prefab/lang/tr_tr.json [hunk 3]
```diff
@@ -116,32 +116,32 @@
...
-    "prefab.gui.building_blueprint": "üreten yapı",
+    "prefab.gui.building_blueprint": "Üretim Taslağı",
     "prefab.gui.starter.tab.general": "Genel",
     "prefab.gui.starter.tab.config": "Seçenekler",
     "prefab.gui.starter.tab.block": "Bloklar/Boyut",
-    "prefab.gui.starter.style.label": "Ev Boyutu",
-    "prefab.gui.starter.floortype.label": "Zemin Blok Tipi",
-    "prefab.gui.starter.ceilingtype.label": "Tavan Blok Tipi",
-    "prefab.gui.starter.walltype.label": "Duvar Odun Tipi",
+    "prefab.gui.starter.style.label": "Zemin Blok Türü",
```
Comment (Brian-Wuest) [inline src/main/resources/assets/prefab/lang/tr_tr.json:129]: Why is this "Ground Block Type" when it's supposed to be "House Size""

**Review by Brian-Wuest — CHANGES_REQUESTED** [review]

**Review by Brian-Wuest — APPROVED** [review]

**Pull request merged**